src/main/java/org/broadinstitute/hellbender/utils/SimpleInterval.java

```diff
@@ -339,4 +340,12 @@ public SimpleInterval expandWithinContig( final int padding, final SAMSequenceDi
 
          return expandWithinContig(padding, contigRecord.getSequenceLength());
      }
+
+     public static SimpleInterval valueOf(final Interval interval) {
```

Comment:
Isn't this redundant with SimpleInterval(Locatable)

Reply:
Ok